### PR TITLE
Add `ls path`

### LINKED ISSUE #1082: Add `spago ls paths` command
We should add a command to display the [common paths used by Spago](https://github.com/purescript/spago/blob/master/src/Spago/Paths.purs):
- the global cache
- the local cache
- the registry repo
- the index
- the database

### PR DESCRIPTION
**Pull request by JordanMartinez**
### Description of the change

Fixes #1082. 

Does this need an flag for each path value?

### Checklist:

- [ ] Added the change to the "Unreleased" section of the changelog
- [ ] Added some example of the new feature to the `README`
- [ ] Added a test for the contribution (if applicable)

**P.S.**: the above checks are not compulsory to get a change merged, so you may skip them. However, taking care of them will result in less work for the maintainers and will be much appreciated 😊

### CODE REVIEW

File: src/Spago/Command/Ls.purs [hunk 1]
```diff
@@ -45,6 +53,18 @@ type LsEnv =
   , selected :: WorkspacePackage
   }
 
+listPaths :: Spago { logOptions :: LogOptions } Unit
+listPaths = do
+  logDebug "Running `listPaths`"
+  traverse_ logInfo
```
Comment (f-f) [inline src/Spago/Command/Ls.purs:59]: `logInfo` goes to stderr, which we use for logging. Program output should go to `stdout`.

We already have facilities for this (the `output` function), see e.g. how the other `ls` commands or the `registry‘ commands do it.
It's quite fancy at this point 😄

We might want a table here, as it's the default output mode for the other ls commands too

File: src/Spago/Command/Ls.purs [hunk 2]
```diff
@@ -45,6 +53,18 @@ type LsEnv =
...
+    [ "Global cache path: " <> Paths.globalCachePath
+    , "Global registry path: " <> Paths.registryPath
+    , "Global registry index path: " <> Paths.registryIndexPath
+    , "Global package sets path: " <> Paths.packageSetsPath
+    , "Local cache path: " <> Paths.localCachePath
+    , "Local cache packages path: " <> Paths.localCachePackagesPath
+    ]
```
Comment (f-f) [inline src/Spago/Command/Ls.purs:66]: Could we add the db path here as well? We'll need to move it from `Spago.Db.databasePath` to here

File: src/Spago/Command/Ls.purs [hunk 3]
```diff
@@ -45,6 +53,18 @@ type LsEnv =
   , selected :: WorkspacePackage
   }
 
+listPaths :: Spago { logOptions :: LogOptions } Unit
+listPaths = do
```
Comment (f-f) [inline src/Spago/Command/Ls.purs:57]:
```suggestion
listPaths { json } = do
```

We don't want flags for each path, but we do want a flag to output Json, as with the other ls commands

**Comment by JordanMartinez** [comment]
Latest commit addressed feedback.

**Review by f-f — APPROVED** [review]
This is great, thanks Jordan!

**Pull request merged**